### PR TITLE
Updated content and links to match current Org settings

### PR DESCRIPTION
**Pull request by rtrieu**
### What does this PR do?
- Updates all Organization Settings docs to match what is currently in the UI.
- Adds/updates links accordingly.

### Motivation
Reach parity with UI and reduce customer confusion: [DOCS-5431](https://datadoghq.atlassian.net/browse/DOCS-5431).

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-5431]: https://datadoghq.atlassian.net/browse/DOCS-5431?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/account_management/org_settings.md [hunk 1]
```diff
@@ -13,80 +13,68 @@ further_reading:
 The Organization Settings section is available to [Administrators][1] by clicking **Organization Settings** from the account menu in the bottom of the left side navigation or by selecting **Organization Settings** from the header dropdown at the top of the Personal Settings page.
 
 Organization Settings allow you to manage users, groups, RBAC, keys, and tokens. This page outlines every section and where in the documentation you can learn about specific tasks in **Organization Settings**.
-### Accounts
+### Identity & Accounts
```
Comment (estherk15) [inline content/en/account_management/org_settings.md:16]:
```suggestion

## Identity & Accounts
```

File: content/en/account_management/org_settings.md [hunk 2]
```diff
@@ -13,80 +13,68 @@ further_reading:
...
+Read the [Teams][3] documentation to manage teams for organizing your assets within Datadog.
 
-#### Roles
+#### Service accounts
 
-To learn about default and custom roles in Datadog, read the [Role Based Access Control documentation][4].
+[Service accounts][4] are non-interactive accounts that you can use to own application keys and other resources that are shared across your teams. Service account application keys can only be viewed once by the individual who created the key. You can use service accounts to access Datadog APIs without associating your application or script with a particular person.
 
-#### SAML Group Mappings
+### Authentication
```
Comment (estherk15) [inline content/en/account_management/org_settings.md:29]:
```suggestion
## Authentication
```

File: content/en/account_management/org_settings.md [hunk 3]
```diff
@@ -13,80 +13,68 @@ further_reading:
...
 
-When enabled, users logging in with SAML to your Datadog account are permanently stripped of their current roles and reassigned to new roles. The SAML assertion passed on from the Identity Provider and the mappings you create determine each user's new roles.
+#### Login methods
 
-Users who log in with SAML and do not have values that map to a Datadog role are permanently stripped of all roles. That user may no longer log in.
-To learn how to create and set mappings, read the [Mapping SAML attributes documentation][5].
+The **Login Methods** tab shows password, Google, and SAML authentication settings. You can toggle each with the **Enabled by Default** dropdowns. In order to be "SAML Strict" or strict for any other type of login, disable the other login method types. You can allow per-user overrides in the User Management tab to allow users to login with another login method if needed.
 
+Read the [Configuring Login Methods][5] documentation to authenticate users to log into your Datadog organization.
 ##### SAML settings
```
Comment (estherk15) [inline content/en/account_management/org_settings.md:36]:
```suggestion

##### SAML settings
```

File: content/en/account_management/org_settings.md [hunk 4]
```diff
@@ -13,80 +13,68 @@ further_reading:
...
 
-#### Client Tokens
+When enabled, users logging in with SAML to your Datadog account are permanently stripped of their current roles and reassigned to new roles. The SAML assertion passed on from the Identity Provider and the mappings you create determine each user's new roles.
 
-Client tokens are used to send events and logs from your user's web and mobile applications. They are unique to your organization. Deleting a client token that is linked to a RUM Application causes your RUM Application to stop reporting. The [process to create client tokens][8] is similar to that for API and application keys.
+Users who log in with SAML and do not have values that map to a Datadog role are permanently stripped of all roles. That user may no longer log in.
+To learn how to create and set mappings, read the [Mapping SAML attributes documentation][7].
 
-#### Events API Emails
+### Access
```
Comment (estherk15) [inline content/en/account_management/org_settings.md:47]:
```suggestion
## Access
```

File: content/en/account_management/org_settings.md [hunk 5]
```diff
@@ -13,80 +13,68 @@ further_reading:
...
+Client tokens are used to send events and logs from your user's web and mobile applications. They are unique to your organization. Deleting a client token that is linked to a RUM Application causes your RUM Application to stop reporting. The [process to create client tokens][11] is similar to that for API and application keys.
 
-The [**OAuth Apps**][10] page allows you to view or manage OAuth applications in your organization.
+#### Events API emails
 
-### General
+If your application does not have an existing Datadog integration, and you don't want to create a custom Agent check, you can send events with email. To learn how to set up events API emails, read the [Events with email guide][12].
 
-#### Rename organization
+### Products
```
Comment (estherk15) [inline content/en/account_management/org_settings.md:73]:
```suggestion
## Products
```

File: content/en/account_management/org_settings.md [hunk 6]
```diff
@@ -95,26 +83,66 @@ Users with `Org Management` permission can enable the out-of-contract retention
...
 
 {{< img src="account_management/monitors-time-zone-preference.png" alt="Monitors Time Zone Preferences" style="width:70%;" >}}
 
 The setting applies to **all** Monitor alert notifications, as it's an org-wide setting.
 
+#### Synthetic tests
+
+Learn how to access and control [Synthetic Monitoring Settings][13].
+
+### Security
```
Comment (estherk15) [inline content/en/account_management/org_settings.md:100]:
```suggestion
## Security
```

File: content/en/account_management/org_settings.md [hunk 7]
```diff
@@ -95,26 +83,66 @@ Users with `Org Management` permission can enable the out-of-contract retention
...
+
+#### Public sharing
+
+The **Public Sharing** tab contains lists of shared dashboards and shared graphs. You can also edit your sharing settings by clicking the **Enabled** toggles.
+
+#### OAuth Apps
+
+The [**OAuth Apps**][14] page allows you to view or manage OAuth applications in your organization.
+
+### Compliance
```
Comment (estherk15) [inline content/en/account_management/org_settings.md:110]:
```suggestion
## Compliance
```

File: content/en/account_management/org_settings.md [hunk 8]
```diff
@@ -95,26 +83,66 @@ Users with `Org Management` permission can enable the out-of-contract retention
...
+#### Audit trail
+
+The **Audit Trail** tab in the Organization Settings page opens a new tab to the Audit Events Explorer.
+
+#### Audit trail settings
+
+The **Audit Trail Settings** tab allows you to set the retention period of audit trails and enable archiving to other cloud storage services.
+
+### General
```
Comment (estherk15) [inline content/en/account_management/org_settings.md:119]:
```suggestion
## General
```

**Review by estherk15 — APPROVED** [review]
Looks good Rosa, left some formatting suggestions!

**Comment by rtrieu** [comment]
@estherk15 Thanks so much for catching the heading stuff! I got so deep into the editing and rearranging, I forgot to look at the right side nav. I updated the smaller H4s to H3s, etc., just FYI.

**Pull request merged**